src/process/status.cr

```diff
@@ -5,12 +5,33 @@ class Process::Status
   def initialize(@exit_status)
   end
 
+  # Returns `true` if the process was terminated by a signal.
+  def signal_exit?
+    # (((signed char) (((status) & 0x7f) + 1) >> 1) > 0)
+    ((LibC::Char.new(@exit_status & 0x7f) + 1) >> 1) > 0
+  end
+
+  # Returns `true` if the process terminated normally.
+  def normal_exit?
+    !signal_exit?
+  end
+
```

Comment:
Specifically I don't think this is correct.  I don't think you can assume !signal_exit means not stopped or continued.